Solutions/Box/Analytic Rules/BoxNewExternalUser.yaml

```diff
@@ -23,16 +23,14 @@ query: |
   | extend corp_domain = extract(@'@(.*)', 1, SourceLogin)
   | extend new_domain = tolower(extract(@'@(.*)', 1, SourceLogin))
   | where corp_domain != new_domain

```

Comment:
same comment above.. Add output formatting, aggregation and additional entitytypes along with UPNDomain